modules/core/src/v2/internal/util.ts

```diff
@@ -197,4 +197,45 @@ export class Util {
     const pubKey = ethUtil.ecrecover(Buffer.from(msgHash, 'hex'), v, r, s);
     return ethUtil.bufferToHex(ethUtil.pubToAddress(pubKey));
   }
+
+  /**
+   * Take the implementation address for the proxy contract, and get the binary initcode for the associated proxy
+   * @param implementationAddress The address of the implementation contract for the proxy
+   * @return {string} Binary hex string of the proxy
+   */
+  static getProxyInitcode = (implementationAddress) => {
+    const target = ethUtil.stripHexPrefix(implementationAddress.toLowerCase()).padStart(40, '0');
+
+    // bytecode of the proxy, from:
+    // https://github.com/BitGo/eth-multisig-v4/blob/d546a937f90f93e83b3423a5bf933d1d77c677c3/contracts/CloneFactory.sol#L42-L56
+    return `0x3d602d80600a3d3981f3363d3d373d3d3d363d73${target}5af43d82803e903d91602b57fd5bf3`;
```

Comment:
the target here should be using rlp encoding instead of what we're doing here

Reply:
I have taken this function from the microservices only, please let me know if it still needs to be changed. https://github.com/BitGo/bitgo-microservices/blob/develop/packages/wallet-platform/app/statics/ethContracts.js#:~:text=*/-,exports.getProxyInitcode%20%3D%20(implementationAddress)%20%3D%3E%20%7B,-const%20target%20%3D%20ethUtil